packages/block-editor/src/components/link-control/link-preview.js

```diff
@@ -79,7 +79,8 @@ export default function LinkPreview( {
 
 	return (
 		<div
-			aria-label={ __( 'Currently selected' ) }
+			role="group"
+			aria-label={ __( 'Manage link' ) }
```

Comment:
Can you help me to understand why you feel the word manage is more appropriate here?

This is a preview of the currently selected link so that's why it's named as such.

Reply:
@getdave, did you read the issue before asking your question? I think I already explained there the reasoning.

Reply:
Yes I read it.

However I did miss the fact that you'd wrapped the _entire_ `LinkPreview` component in a `Manage link` group. That was an oversight on my part which I was happy to correct.